docs/serving/samples/cloudevents/cloudevents-rust/index.md

```diff
@@ -5,4 +5,163 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```

Comment:
cc @RichieEscarez 
Do we need to use the index+README for some reason or is it OK to just add the content to index like this?

Reply:
@mattmoor removed it here https://github.com/knative/docs/pull/2805

Reply:
@abrennan89 I think Github prefers README, but the `readfile` shortcode doesn't work with nested shortcodes IIUC